src/rpc/mempool.cpp

```diff
@@ -968,7 +968,7 @@ static RPCHelpMan submitpackage()
                                 {{RPCResult::Type::STR_HEX, "", "transaction wtxid in hex"},
                             }},
                         }},
-                        {RPCResult::Type::STR, "error", /*optional=*/true, "The transaction error string, if it was rejected by the mempool"},
+                        {RPCResult::Type::STR, "error", /*optional=*/true, "Error string if rejected from mempool, or \"package-not-validated\" when the package aborts before any per-tx processing."},
```

Comment:
is optional now false?

Reply:
I'd think it stays optional, as successful (and partially successful) submissions won't have an error for every tx. See, eg, [this test](https://github.com/bitcoin/bitcoin/blob/edb871cba22a658714ce97e843ff84cfe1ba9ac4/test/functional/rpc_packages.py#L417), where only the child has an error.

Reply:
Oh I did not read that carefully 🤦 yes.